examples/src/components/GeoPushpinChartConfigurationLegendExample.jsx

```diff
@@ -0,0 +1,46 @@
+// (C) 2007-2020 GoodData Corporation
```

Comment:
This is new, so It should be // (C) 2020 GoodData Corporation

Reply:
updated